packages/icons/icons.json

```
@@ -2193,406 +2193,406 @@
     },
     {
         "displayName": "Join table",
         "iconName": "join-table",
-        "tags": "combine, attach, connect, link, unite",
+        "tags": "combine, attach, connect, link, unite, merge data, excel, calendars, dataframe, tables, sheets, intersection",
         "group": "table",
         "codepoint": 61963
     },
     {
         "displayName": "Derive column",
         "iconName": "derive-column",
-        "tags": "table, obtain, get, take, develop",
+        "tags": "table, obtain, develop, function container, function in a box",
         "group": "action",
         "codepoint": 61815
     },
     {
         "displayName": "Rotate image: left",
         "iconName": "image-rotate-left",
-        "tags": "picture, turn, alternate",
+        "tags": "picture, turn, alternate, media, landscape",
         "group": "media",
         "codepoint": 61944
     },
     {
         "displayName": "Rotate image: right",
         "iconName": "image-rotate-right",
-        "tags": "picture, turn, alternate",
+        "tags": "picture, turn, alternate, media, landscape",
         "group": "media",
         "codepoint": 61945
     },
     {
         "displayName": "Known vehicle",
         "iconName": "known-vehicle",
-        "tags": "car, automobile, vehicle, van, drive, ride, distance, navigation, directions",
+        "tags": "car, automobile, van, drive, ride, distance, navigation, directions, approved, checked",
         "group": "interface",
         "codepoint": 61974
     },
     {
         "displayName": "Unknown vehicle",
         "iconName": "unknown-vehicle",
-        "tags": "car, automobile, vehicle, van, drive, ride, distance, navigation, directions",
+        "tags": "car, automobile, van, drive, ride, distance, navigation, directions, missing, mystery, question mark",
         "group": "interface",
         "codepoint": 62212
     },
     {
         "displayName": "Scatter plot",
         "iconName": "scatter-plot",
-        "tags": "graph, diagram",
+        "tags": "graph, diagram, data",
         "group": "data",
         "codepoint": 62110
     },
     {
         "displayName": "Oil field",
         "iconName": "oil-field",
-        "tags": "fuel, petroleum, gas, well, drilling, pump",
+        "tags": "fuel, petroleum, gas, well, drilling, pump, wheel, hammer, engineering, refinery, well",
         "group": "interface",
         "codepoint": 62051
     },
     {
         "displayName": "Rig",
         "iconName": "rig",
-        "tags": "fuel, petroleum, gas, well, drilling",
+        "tags": "fuel, petroleum, gas, well, drilling, offshore, asset, flare, torch, oil, candle man",
         "group": "interface",
         "codepoint": 62100
     },
     {
         "displayName": "New map",
         "iconName": "map-create",
-        "tags": "map, location, position, geography, world",
+        "tags": "map, location, position, geography, world, new brochure, new pamphlet, navigation, travel, gps, coalesce",
```

"coalesce" feels like a stretch here

```suggestion
        "tags": "map, location, position, geography, world, new brochure, new pamphlet, navigation, travel, gps",
```

---

agreed